modules/test/playwright/tests/frontend-data-set-views-web/pages/data_set/tabs/FiltersPage.ts

```diff
@@ -5,47 +5,91 @@
 
 import {Locator, Page, expect} from '@playwright/test';
 
-import {ISelectionFilter} from '../../../utils/types';
+import {IDateRangeFilter, ISelectionFilter} from '../../../utils/types';
 import {DataSetPage} from '../DataSetPage';
 
+interface NewFilterModal {
+	cancelButton: Locator;
+	closeButton: Locator;
+	filterByDropdown: Locator;
+	filterBySelect: Locator;
+	formFeedback: Locator;
+	modalBody: Locator;
+	nameInput: Locator;
+	saveButton: Locator;
+}
+
+interface NewSelectionFilterModal extends NewFilterModal {
+	filterModeRadioButtons: Locator;
+	nameInput: Locator;
+	newFilterModalheading: Locator;
```

Comment:
I think the `heading` should be differentiated for every type of filter. This one could be `newSelectionFilterModalHeading` and we could create `newDateRangeFilterModalHeading` for DateRange below

Reply:
This is not needed, I've removed it